components/selectors/parser.rs

```diff
@@ -2060,7 +2211,10 @@ pub mod tests {
 
     #[test]
     fn test_universal() {
-        let selector = &parse("*|*::before").unwrap().0[0];
+        let selector = &parse_ns("*|*::before", &DummyParser {
+            default_ns: Some(DummyAtom::from("https://mozilla.org")),
```

Comment:
nit: Maybe having a `DummyParser::default()` or something like that would be nice, to avoid some duplication here.